### ISSUE TITLE
How can I download the raw image data?

### ISSUE THREAD

**Issue by PhoebusSi** [haotian-liu/LLaVA#13]
How can I download the raw image data?

**Comment by haotian-liu** [comment]
You can download the images from COCO website: https://cocodataset.org/#download

**Comment by PhoebusSi** [comment]
Is it enough for me to download these two: "2014 Train images [83K/13GB]" and "2014 Val images [41K/6GB]"?

**Comment by PhoebusSi** [comment]
Do I need to download the 2017 version of COCO ?

**Comment by haotian-liu** [comment]
We used COCO 2014 to build the instruction tuning dataset, so COCO 2014 is sufficient

**Comment by PhanTask** [comment]
> We used COCO 2014 to build the instruction tuning dataset, so COCO 2014 is sufficient

Hi, does it mean in the current finetune command `--image_folder /Data/haotian/coco/train2017` actually should be the image folder of train2014?

**Comment by haotian-liu** [comment]
@PhanTask Hi sorry for the confusion.  It should be train2014.  I have changed that in the command line.

**Comment by PhanTask** [comment]
> @PhanTask Hi sorry for the confusion. It should be train2014. I have changed that in the command line instruction. Thanks.

Thanks for looking into it, Haotian! Yesterday I launched finetuning with COCO train2017 images and it seems to be no issues (still running with reduced loss). The image_id in the 150K instruction set indicates they might come from train2017 (e.g., no GCC-train- prefix as seen in train2014). If I launch training with train2014, it reports file name missing issue. Could you please confirm if the finetuning images are actually from train2017? Thanks.

**Comment by haotian-liu** [comment]
Hi @PhanTask the images are coming from COCO 2014, but I may have renamed the file due to some historic reasons.  The COCO 2017 training image is a super set of COCO 2014, so you will not notice any issues when training.  The current set is only using COCO 2014 training images.  This makes sure that we can use the coco 2014 validation set for evaluation.

I may modify the code this week, so that it automatically recognizes the COCO 2014 naming convention, so that it will not pose errors when you use either of 2014/2017 split.

**Comment by PhanTask** [comment]
> Hi @PhanTask the images are coming from COCO 2014, but I may have renamed the file due to some historic reasons. The COCO 2017 training image is a super set of COCO 2014, so you will not notice any issues when training. The current set is only using COCO 2014 training images. This makes sure that we can use the coco 2014 validation set for evaluation.
> 
> I may modify the code this week, so that it automatically recognizes the COCO 2014 naming convention, and it will not raise errors when you use either of 2014/2017 split.

Great, thanks!
